app/src/main/cpp/skyline/applet/web_applet.cpp

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2022 Skyline Team and Contributors (https://github.com/skyline-emu/)
+// Copyright © 2020 yuzu Emulator Project (https://github.com/yuzu-emu/)
+
+#include <input.h>
+#include <input/npad.h>
+#include <services/applet/common_arguments.h>
+#include <services/am/storage/ObjIStorage.h>
+#include "offline_web_applet.h"
```

Comment:
I think this was supposed to be "web_applet.h" or the file name wasn't updated to match.